cg/cli/upload/scout.py

```diff
@@ -52,18 +54,26 @@ def create_scout_load_config(context, case_id: str, print_console: bool, re_uplo
     LOG.info("Fetching family object")
     case_obj: Family = status_api.family(case_id)
     LOG.info("Create load config")
-    scout_load_config: ScoutLoadConfig = scout_upload_api.generate_config(case_obj.analyses[0])
+    try:
+        scout_load_config: ScoutLoadConfig = scout_upload_api.generate_config(case_obj.analyses[0])
```

Comment:
why not have a check?
if not case_obj.analyses:
    LOG.bla()
    raise click.Abort

?

Reply:
oki